core/src/test/scala/akka/persistence/cassandra/ReconnectSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra
+
+import java.io.File
+
+import akka.actor.{ ActorSystem, Props }
+import akka.persistence.cassandra.CassandraLifecycle.AwaitPersistenceInit
+import akka.persistence.cassandra.testkit.CassandraLauncher
+import akka.testkit.{ ImplicitSender, SocketUtil, TestKit, TestKitBase }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.Suite
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object ReconnectSpec {
+  val freePort = SocketUtil.temporaryLocalPort()
+  val config = ConfigFactory.parseString(s"""
+      datastax-java-driver {
+        basic.load-balancing-policy.local-datacenter = "datacenter1"
+        // Will fail without this setting 
+        advanced.reconnect-on-init = true      
```

Comment:
right, that's documented as an important property.
could you also add a comment about it in our reference.conf.

@ennru this property should be recommended in Alpakka docs also, if it's not already

Reply:
Thanks, I added https://github.com/akka/alpakka/issues/2218

Reply:
Yes i'll add it to reference